mavis/test/data/__init__.py

```diff
@@ -152,6 +155,10 @@ def create_file_from_template(
         if self.nurse:
             static_replacements["<<NURSE_EMAIL>>"] = self.nurse.username
 
+        if self.clinic:
```

Comment:
I think this is being passed in as a list, in which case we can iterate over the list to generate the index, similar to the schools.

```py
for index, clinic in enumerate(self.clinics):
    static_replacements[f"<<CLINIC_{index}_NAME>>"] = clinic.name
    static_replacements[f"<<CLINIC_{index}_NAME_LOWER>>"] = clinic.name.lower()
```